src/main/java/club/minnced/discord/webhook/send/component/Button.java

```diff
@@ -0,0 +1,166 @@
+package club.minnced.discord.webhook.send.component;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.json.JSONObject;
+import org.json.JSONPropertyIgnore;
+
+import java.net.URL;
+
+public class Button implements ActionComponent {
+
+    private final ButtonStyle style;
+    private final String label, customID, url;
+    private final boolean disabled;
+    private PartialEmoji emoji;
+
+    public Button(@NotNull ButtonStyle style, @NotNull String label, @Nullable String customID, @Nullable String url, boolean disabled) {
+        this.style = style;
+        this.label = label;
+        this.customID = customID;
+        this.url = url;
+        this.disabled = disabled;
+    }
+
+    /**
+     * A primary style button with the provided id and label
+     * @param customID dev-defined id
+     * @param label text on button
+     * @return A primary style button with the provided id and label
```

Comment:
This is the format I would like to see:

```suggestion
     * A button with style set to {@link ButtonStyle#PRIMARY}
     *
     * @param  customID
     *         Custom id used for handling of interactions
     * @param  label
     *         Label used to display text on a button
     *
     * @return A primary style button with the provided id and label
```